build.gradle.kts

```diff
@@ -27,10 +27,10 @@ repositories {
 }
 
 dependencies {
-    compileOnly("org.jetbrains.kotlin:kotlin-gradle-plugin:1.9.20")
-    compileOnly("com.android.tools.build:gradle:8.0.2")
-    compileOnly("org.jetbrains.intellij.plugins:gradle-intellij-plugin:1.15.0")
-    compileOnly("org.jetbrains.kotlinx:binary-compatibility-validator:0.13.2")
+    compileOnly("org.jetbrains.kotlin:kotlin-gradle-plugin:1.9.23")
+    compileOnly("com.android.tools.build:gradle:8.3.1")
```

Comment:
Do we really need to update AGP and Gradle now? I'm trying to keep versions consistent across all projects that use this plugin.

Reply:
Ok 🙂

Reply:
I'm using IntelliJ IDEA, I guess we could update Gradle to 8.4 and AGP to 8.2.0.

Reply:
Done